packages/core/client/src/schema-component/antd/icon-picker/IconPicker.tsx

```diff
@@ -11,28 +11,45 @@ import { CloseOutlined, LoadingOutlined } from '@ant-design/icons';
 import { useFormLayout } from '@formily/antd-v5';
 import { connect, mapProps, mapReadPretty } from '@formily/react';
 import { isValid } from '@formily/shared';
-import { Button, Space } from 'antd';
+import { Button, Space, Input } from 'antd';
 import React, { useState } from 'react';
 import { useTranslation } from 'react-i18next';
 import { Icon, hasIcon, icons } from '../../../icon';
 import { StablePopover } from '../popover';
+import { debounce } from 'lodash';
+
+const { Search } = Input;
 
 export interface IconPickerProps {
   value?: string;
   onChange?: (value: string) => void;
   disabled?: boolean;
   suffix?: React.ReactNode;
+  iconSize?: number;
+  searchable?: boolean;
 }
 
 interface IconPickerReadPrettyProps {
   value?: string;
 }
 
 function IconField(props: IconPickerProps) {
+  const availableIcons = [...icons.keys()];
   const layout = useFormLayout();
-  const { value, onChange, disabled } = props;
+  const { value, onChange, disabled, iconSize = 24, searchable = true } = props;
```

Comment:
For default `iconSize`, it might be better to use a font size token like `fontSizeXL` instead of a fixed number so that it could be more comfortable when user changing the font size of the full site via theme editor.

```
import { theme } from 'antd';

// ...
const { token } = theme.useToken();
token.fontSizeXL
```

Reference: https://ant.design/docs/react/customize-theme/#design-token

Reply:
In this case I'd prefer `fontSizeHeading3`, since even the xlarge fontsize is too small when dealing with a bunch of icons side-by-side.

Reply:
It is fine to try `fontSIzeHeading3` to compare the effect, I just gave an example.
And I think the divider could be removed? If it is required, there is also a component you can use. https://ant.design/components/divider